app/src/main/res/values/strings.xml

```diff
@@ -207,5 +207,7 @@ Tap this message (or hit back) to skip this step.</string>
   <string name="give_permission">Give permission</string>
   <string name="use_external_storage">Use external storage</string>
   <string name="use_external_storage_summary">Save pictures taken with the in-app camera on your device</string>
-    <string name="login_to_your_account">Login to your account</string>
+  <string name="send_log_file">Send Log file</string>
```

Comment:
Sorry for the tiny detail, but how about "Send log file"? :-)

Reply:
Ha ha, that capital L was bothering me too! Go ahead and review the rest. Once you are done I will make all the requested changes together and push them.